src/Media/Media.ts

```diff
@@ -1,10 +1,10 @@
 import { Location } from '../Geo';
 
 export interface MainMedia {
-	square: Medium;
-	videoPreview?: Medium;
-	portrait: Medium;
-	landscape: Medium;
+	square: Medium | null;
```

Comment:
it's probably because of strict null checks, still, square previouw should be never null, or?